src/onnx/parse_gemm.cpp

```diff
@@ -71,6 +71,14 @@ struct parse_gemm : op_parser<parse_gemm>
 
         std::vector<int64_t> perm = {1, 0};
         auto dot_type             = a_arg->get_shape().type();
+
+        a_arg = (trans_a)
```

Comment:
I do not see why this would necessary

Reply:
This would make `mul` operation take transposed inputs in pointwise kernel. It may impact performance ( i am 100% sure).